lib/utils.js

```diff
@@ -1,52 +1,122 @@
-'use strict';
+'use strict'
 
-var Cache = require('./cache');
-var Mustache = require('mustache');
-var Needle = require('needle');
-var Winston = require('winston');
+const Cache = require('./cache')
+const Mustache = require('mustache')
+const Needle = require('needle')
+const Winston = require('winston')
+const CronJob = require('cron').CronJob
+const Routes = require('./routes')
+const URL = require('url')
+const removeRoute = require('express-remove-route')
```

Comment:
I'm hesitant of including random modules like this. Looking at the GitHub repo it doesn't seem to be super solid, and I think that we'd be better of implementing the functionality ourselves. It doesn't need to be as complicated as they have made it for our use case.